### PR TITLE
add informational EIP for DAO HF

### PR DESCRIPTION
**Pull request by cdetrio**
This completes the HF table https://github.com/ethereum/EIPs/pull/676

### CODE REVIEW

File: EIPS/eip-779.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+This documents the changes included in the hard fork named "DAO Fork". Unlike other hard forks, the DAO Fork did not change the protocol; all EVM opcodes, transaction format, block structure, and so on remained the same. Rather, the DAO Fork was an "irregular state change" that transferred ether balances from a list of accounts ("child DAO" contracts) into a specified account (the "WithdrawDAO" contract).
+
+## Specification
+
+- Codename: DAO Fork
+- Activation:
+  - Block == 1,920,000 on Mainnet
+
+For the full specification, see references [1] and [2].
```
Comment (nicksavers) [inline EIPS/eip-779.md]: Probably better to formalize it in the EIP itself

* The DAO (`0xbb9bc244d798123fde783fcc1c72d3bb8c189413`)
* its extraBalance (`0x807640a13483f8ac783c557fcdf27be11ea4ac7a`)
* all children of the DAO creator (`0x4a574510c7014e4ae985403536074abe582adfc8`) 
* and the extraBalance of each child are encoded into a list `L` at block 1880000.

At the beginning of block 1920000, all ether throughout all accounts in `L` will be transferred to contract account `C`, which is deployed at `0xbf4ed7b27f1d666546e30d74d50d173d20bca754`.

```
// Deployed on mainnet at 0xbf4ed7b27f1d666546e30d74d50d173d20bca754

contract DAO {
    function balanceOf(address addr) returns (uint);
    function transferFrom(address from, address to, uint balance) returns (bool);
    uint public totalSupply;
}

contract WithdrawDAO {
    DAO constant public mainDAO = DAO(0xbb9bc244d798123fde783fcc1c72d3bb8c189413);
    address public trustee = 0xda4a4626d3e16e094de3225a751aab7128e96526;

    function withdraw(){
        uint balance = mainDAO.balanceOf(msg.sender);

        if (!mainDAO.transferFrom(msg.sender, this, balance) || !msg.sender.send(balance))
            throw;
    }

    function trusteeWithdraw() {
        trustee.send((this.balance + mainDAO.balanceOf(this)) - mainDAO.totalSupply());
    }
}
```

Compiled with Solidity version `v0.3.5-2016-07-01-48238c9`

File: EIPS/eip-779.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+## Preamble
+
+    EIP: 779
+    Title: Hardfork Meta: DAO Fork
+    Author: Casey Detrio
+    Type: Meta
```
Comment (nicksavers) [inline EIPS/eip-779.md]: @cdetrio Should this be Informational?

Comment (cdetrio) [reply EIPS/eip-779.md]: yes, thanks

**Comment by pirapira** [comment]
Shall we also add this one into "Finalized EIPs" in README?

**Comment by nicksavers** [comment]
@pirapira It will be part of https://github.com/ethereum/EIPs/pull/676 where those specific informational EIPs are moved to a hard fork table

**Comment by pirapira** [comment]
@nicksavers I see. Thanks!

**Pull request merged**